src/pages/registration/registration-stats-page.js

```diff
@@ -70,6 +70,13 @@ const RegistrationStatsPage = ({currentSummit, summitStats, match, getRegistrati
 
   const totalAttendees = totalRealAttendees + totalVirtualAttendees;
 
+  const toDecimalMark = num => { 
```

Comment:
instead of creating this helper function , could be use this
https://github.com/fntechgit/summit-registration-lite/blob/main/src/helpers/formatCurrency.js 
see https://developer.mozilla.org/es/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat for all money values ?
regards

Reply:
done